app/uk/gov/hmrc/teamsandrepositories/connectors/JenkinsConnector.scala

```diff
@@ -74,38 +72,30 @@ class JenkinsConnector @Inject()(
     httpClientV2
       .post(url)
       .setHeader("Authorization" -> authorizationHeader)
-      .execute[JenkinsBuildData]
+      .execute[BuildData](readFromJson(BuildData.jenkinsReads, implicitly[Manifest[BuildData]]), ec)
```

Comment:
can we not just add an `implicit val jr = BuildData.jenkinsReads` in this function?